packages/shared-models/src/lib/status-setting.ts

```diff
@@ -0,0 +1,27 @@
+import { TaskStatus } from "@prisma/client"
+import { taskStatusModal } from "./_prisma"
+import { ObjectId } from "bson"
+
+export const mdStatusSettingAdd = async (data: Omit<TaskStatus, 'id'>) => {
+
+	const id = new ObjectId().toString()
+	return taskStatusModal.create({
+		data: {...data, ...{id}}
+	})
+}
+
+export const mdStatusSettingGetAll = async (projectId: string) => {

```

Comment:
đặt lại toàn bộ tên hàm đi nhé, lấy `mdStatus` làm tiền tố thôi, bỏ cái đoạn Setting đi

cơ bản cách đặt tên hàm cũng na ná đặt tên file

ở đây ta chỉ cần các tên hàm mô tả hành động đối với model là được, ko cần gắn cụ thể chữ Setting vào làm gì